test/ExtendedXmlSerializerTest/TestObject/TestClassCollection.cs

```diff
@@ -0,0 +1,28 @@
+﻿// MIT License
+// 
+// Copyright (c) 2016 Wojciech Nagórski
```

Comment:
Coooooool. :D I will think about this.  I am not so keen on code file copyrights (the less words in the code, the better, IMO). I would be more apt to have a mention in the Wiki, if anything.  GitHub automatically keeps track of contributors, right?  That's good enough for me.  The most important thing is code works and tests pass!